install/upgrade.go

```diff
@@ -81,7 +88,55 @@ func (k *K8sInstaller) Upgrade(ctx context.Context) error {
 		patched++
 	}
 
-	if patched > 0 && k.params.Wait {
+	hubbleDeployment, err := k.client.GetDeployment(ctx, k.params.Namespace, defaults.RelayDeploymentName, metav1.GetOptions{})
+	if err != nil {
+		k.Log("ℹ️  Unable to retrieve Deployment of hubble-relay")
+		message++
```

Comment:
Incrementing `message` has no effect here, as any new value is overwritten at line 97.